### DISCUSSION TITLE
Is there any way to sort pipelines in GoCD by update time?

### DISCUSSION THREAD

**Discussion by zszen** [gocd/gocd#12642] [Q&A] [answered]
This is a practical function. Sometimes, there are too many pipelines, and it is easy to find which ones have been updated recently by sorting them by time.

**Comment by chadlwilson** [comment, marked as answer]
Unfortunately not.

Personally I tend to use a mixture of groups (if permissions are relevant) and user-specific filter views to focus on the pipelines of relevance to a task, and then just use the browser search to search for the text of the dates in the last trigger times.

**Reply by zszen** [reply]
okey
